### DISCUSSION TITLE
How to enter the container terminal using `docker exec -it`?

### DISCUSSION THREAD

**Discussion by osaxma** [PostgREST/postgrest#2149] [Q&A] [answered]
I'm unable to access `bash` or `shell` using the following commands (image:  `postgrest/postgrest:v9.0.0`):

```console
osaxma:~$ docker exec -it <container_id> bin/bash
OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "bin/bash": stat bin/bash: no such file or directory: unknown

osaxma:~$ docker exec -it <container_id> bin/sh
OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "bin/sh": stat bin/sh: no such file or directory: unknown
```

I noticed in `docker inspect`that the only available command is `bin/postgrest` but when browsing the issues here, I've seen examples where `docker exec -it <container_id> bin/bash` was used.

**Comment by wolfgangwalther** [comment, marked as answer]
Those examples were all pre-v8.0. In v8.0 we changed the way we build our docker images to a minimal image:
https://postgrest.org/en/stable/releases/v8.0.0.html#changed

**Reply by osaxma** [reply]
Thanks @wolfgangwalther for the quick response. 

I'm having trouble updating the config (mainly `db-schemas`) of a running PostgrREST container. I'm using a tool which spins up the container as part of an app (i.e. supabase cli) but doesn't provide an option set the environment variables. 

Is there any way or workaround to update the config in such case? 

This is for local development only and the goal is to match the remote hosted environment which has a UI to update the `db-schemas` but the cli for local dev does not have an option.

**Reply by wolfgangwalther** [reply]
Can you use [in-database configuration](https://postgrest.org/en/stable/configuration.html#in-database-configuration) instead?

This will allow to change the `db-schemas` without restart. Just send a notification from PG: https://postgrest.org/en/stable/configuration.html#in-database-configuration-reloading

**Reply by osaxma** [reply]
> Can you use [in-database configuration](https://postgrest.org/en/stable/configuration.html#in-database-configuration) instead?

I tried that but none of the `pgrst` configs were part of the database container (for supabase -- It looks like they've a different setup).

**Reply by wolfgangwalther** [reply]
No idea about the supabase setup. Maybe ask in one of their repos. Maybe @steve-chavez can chime in, too.
